frontend/src/scenes/experiments/Experiments.tsx

```diff
@@ -74,13 +81,44 @@ export function Experiments(): JSX.Element {
                 )
             },
         },
+        {
+            width: 0,
+            render: function Render(_, experiment: Experiment) {
+                return (
+                    <More
+                        overlay={
+                            <>
+                                <LemonButton type="stealth" to={urls.experiment(`${experiment.id}`)} compact fullWidth>
+                                    View
+                                </LemonButton>
+                                <LemonSpacer />
+                                <LemonButton
+                                    type="stealth"
+                                    style={{ color: 'var(--danger)' }}
+                                    onClick={() =>
+                                        deleteWithUndo({
```

Comment:
Q: Do we support undos everywhere? This seems like an annoying thing to get right, since deleting an experiment implies deleting the corresponding FFs.

Obviously doable, but pollutes the `deleteWithUndo` interface, and can lead to annoying edge cases where one undo succeeds but the other doesn't, and then your experiment is in an inconsistent state.

I'd rather not support it right now, if we can.

Reply:
sure